test/edu/wright/cs/raiderplanner/model/TimetableEventTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2018 - Adam Cone
+ *
+ *
+ *
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package edu.wright.cs.raiderplanner.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Created by Adam Cone 11/14/2018.
+ */
+public class TimetableEventTest {
+
+	String date;
```

Comment:
Though this is a unit test and not production code, these class members should still be private (instead of package visible).

Reply:
I am having a good deal of problems with this change request.  If I make the class members private, I am no long able to run jUnits tests directly against them.  According to my research, the only options I have for testing private functions is to use a nested test class or reflection but this seems overly complicated for our unit tests.  For now, I am requesting that they remain public for the purpose of code readability and simplicity.